src/test/java/vention/pagesTest/BookFlowTest.java

```diff
@@ -0,0 +1,46 @@
+package vention.pagesTest;
+
+import vention.configLoader.ConfigLoader;
+import vention.driver.DriverManager;
+import vention.pages.CookiePopup;
+import vention.pages.HomePage;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import org.testng.Assert;
+
+public class BookFlowTest extends BaseTest {
+  private CookiePopup cookiePopup;
+  private HomePage homePage;
+
+  @BeforeMethod
+  public void beforeMethod() {
+    homePage = new HomePage();
+    homePage.openPage();
+
+    cookiePopup = new CookiePopup();
+    cookiePopup.clickRejectAllButton();
+  }
+
+  @Test
+  public void testBookFlight() {
+    SoftAssert softAssert = new SoftAssert();
+    softAssert.assertTrue(homePage.isSelectFareDisplayed(), "Fare dropdown should be visible");
+    softAssert.assertTrue(homePage.isFromInputDisplayed(), "From input should be visible");
+    softAssert.assertTrue(homePage.isToInputDisplayed(), "To input should be visible");
+    softAssert.assertTrue(homePage.isDepartureDateButtonDisplayed(), "Departure date button should be visible");
+    softAssert.assertTrue(homePage.isFindFlightsButtonDisplayed(), "Find flights button should be visible");
+    softAssert.assertAll();
+
+    homePage.selectFare("oneWay");
```

Comment:
Let's not hardcode test data as strings. Please avoid the cases where a string is hardcoded. 
In such cases it's better to create a constant variable and store the needed info there.

Why? If anything gets changes in the string value, in case of constant variable you will have to update that just once in the variable, in case of hardcoded data - in every place it's used.